### PR TITLE
[hi] Localize site-reliabilty-engineering.md

### LINKED ISSUE #1130: [hi] Localize term `site_reliability_engineering` into `Hindi`
localization of `site_reliabilty_engineering` into Hindi
  

-   https://glossary.cncf.io/site-reliability-engineering/

### PR DESCRIPTION
**Pull request by taniaduggal**
Fixes #1130 
Signed-off-by: Tania Duggal <taniaduggal60@gmail.com>

### CODE REVIEW

**Comment by taniaduggal** [comment]
@anubha-v-ardhan  Please take a look.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: content/hi/site_reliability_engineering.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: साइट विश्वसनीयता इंजीनियरिंग(Site Reliability Engineering)
+status: Completed
+category: संकल्पना
+---
+
+## यह क्या है
+
+साइट विश्वसनीयता इंजीनियरिंग या एसआरई एक अनुशासन है जो संचालन और सॉफ्टवेयर इंजीनियरिंग को जोड़ती है। उत्तरार्द्ध विशेष रूप से बुनियादी ढांचे और संचालन की समस्याओं पर लागू होता है। मतलब, उत्पाद सुविधाओं के निर्माण के बजाय, साइट विश्वसनीयता इंजीनियर अनुप्रयोगों को चलाने के लिए सिस्टम बनाते हैं। इसकी DevOps के साथ समानताएं हैं, लेकिन जब DevOps उत्पादन के लिए कोड प्राप्त करने पर ध्यान केंद्रित करता है, SRE यह सुनिश्चित करता है कि उत्पादन में चल रहा कोड ठीक से काम करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/site_reliability_engineering.md:9]:
```suggestion
साइट विश्वसनीयता इंजीनियरिंग या SRE एक अनुशासन है जो संचालन (operations) और सॉफ्टवेयर इंजीनियरिंग को जोड़ती है। उत्तरार्द्ध विशेष रूप से इंफ्रास्टक्टर और संचालन की समस्याओं पर लागू होता है। अर्थात, उत्पाद सुविधाओं के निर्माण के बजाय, साइट विश्वसनीयता इंजीनियर एप्लीकेशनों को चलाने के लिए सिस्टम बनाते हैं। इसकी डेवॉप्स के साथ समानताएं हैं, लेकिन जहाँ डेवॉप्स उत्पादन के लिए कोड प्राप्त करने पर ध्यान केंद्रित करता है, SRE यह सुनिश्चित करता है कि उत्पादन में चल रहा कोड ठीक से काम करता है।
```

File: content/hi/site_reliability_engineering.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## समस्या
+
+अनुप्रयोगों को मज़बूती से चलाना सुनिश्चित करने के लिए प्रदर्शन की निगरानी, ​​​​अलर्टिंग, डिबगिंग से लेकर समस्या निवारण तक कई क्षमताओं की आवश्यकता होती है। इसके बिना, सिस्टम ऑपरेटर केवल समस्याओं पर प्रतिक्रिया कर सकते हैं बनाम उनसे बचने की दिशा में सक्रिय रूप से काम कर रहे हैं - डाउनटाइम केवल समय की बात बन जाता है।
```
Comment (anubha-v-ardhan) [inline content/hi/site_reliability_engineering.md:13]:
```suggestion
एप्लीकेशनों की विश्वसनीय चलन सुनिश्चित करने के लिए प्रदर्शन की निगरानी, ​​​​अलर्टिंग, डिबगिंग से लेकर समस्या निवारण तक कई क्षमताओं की आवश्यकता होती है। इसके बिना, सिस्टम ऑपरेटर केवल समस्याओं पर प्रतिक्रिया कर सकते हैं बनाम उनसे बचने की दिशा में सक्रिय रूप से काम करना जिससे डाउनटाइम केवल समय की बात बन जाता है।
```

File: content/hi/site_reliability_engineering.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## समाधान
+
+एक एसआरई दृष्टिकोण अंतर्निहित प्रणाली में लगातार सुधार करके सॉफ्टवेयर विकास प्रक्रिया की लागत, समय और प्रयास को कम करता है। सिस्टम लगातार बुनियादी ढांचे और अनुप्रयोग घटकों को मापता है और उनकी निगरानी करता है। जब कुछ गलत हो जाता है, तो सिस्टम साइट विश्वसनीयता इंजीनियरों को इंगित करता है कि इसे कब, कहाँ और कैसे ठीक किया जाए। यह दृष्टिकोण परिचालन कार्यों को स्वचालित करके अत्यधिक स्केलेबल और विश्वसनीय सॉफ्टवेयर सिस्टम बनाने में मदद करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/site_reliability_engineering.md:17]:
```suggestion
एक SRE दृष्टिकोण अंतर्निहित प्रणाली में लगातार सुधार करके सॉफ्टवेयर विकास प्रक्रिया की लागत, समय और प्रयास को कम करता है। सिस्टम लगातार बुनियादी ढांचे और एप्लीकेशन घटकों को मापता है और उनकी निगरानी करता है। जब कुछ गलत हो जाता है, तो सिस्टम साइट विश्वसनीयता इंजीनियरों को इंगित करता है कि इसे कब, कहाँ और कैसे ठीक किया जाए। यह दृष्टिकोण परिचालन कार्यों को स्वचालित करके अत्यधिक स्केलेबल और विश्वसनीय सॉफ्टवेयर सिस्टम बनाने में मदद करता है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you @taniaduggal, please go through the suggested changes and commit them if you agree :)

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Hey @taniaduggal, Can you please change the file name to `site-reliability-engineering.md`. We prefer keeping the file names same as English version

**Comment by taniaduggal** [comment]
@anubha-v-ardhan Please again review at pr. I made suggested changes.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
@taniaduggal The file name is still `site_reliability_engineering.md`, Can we change it to `site-reliability-engineering.md` to maintain consistency with English version.

**Comment by taniaduggal** [comment]
@anubha-v-ardhan How can I change it? Please suggests.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Garima-Negi** [comment]
Hi @taniaduggal , in your local branch `taniaduggal:tania/site_reliability_engineering1` , using an IDE you can select filename and then right click and select rename .

**Review by sayantani11 — APPROVED** [review]
Thanks!
LGTM

**Comment by taniaduggal** [comment]
@Garima-Negi Please merge this PR if changes looks good.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM

**Comment by anubha-v-ardhan** [comment]
Thank you @taniaduggal :)

**Pull request merged**